src/main.js

```diff
@@ -358,6 +384,18 @@ function getArgvConfigPath() {
 	return path.join(os.homedir(), dataFolderName, 'argv.json');
 }
 
+async function getCrashReporterId() {
```

Comment:
I am not a big fan of having to read another file on startup. As I suggested initially, why not simply write this into `argv.json`? E.g. we could have a crash reporter related key in that file that is:
* `false` when disabled
* `UUID` when enabled and used as crash reporter ID
* `undefined` when we first startup with these changes and need to write a new UUID into

Reply:
there is now `enable-crash-reporter` which will take those 3 states and be written to argv.json

* `"enable-crash-reporter": false` which users can write
* `"enable-crash-reporter": <UUID>` when the key is undefined we create a UUID and write to the file, otherwise the value will be used as crash reporter id for any future calls.